src/ocean/text/convert/Integer.d

```diff
@@ -30,6 +30,138 @@ import ocean.transition;
 
 import ocean.core.Traits;
 
+import ocean.core.array.Search : find;
+
+import ocean.util.ReusableException;
+
+import ocean.math.Math;
+
+
+/*******************************************************************************
+
+    Parses a floating point number represented as a string directly to an
+    integer value.
+
+    To represent the fractional part we multiply the value by the requested
+    amount of decimal points and add it up. For example:
+
+    "1.123" -> 1123  (decimal_points = 3)
+    "0.01"  ->   10  (decimal_points = 3)
+
+    Any characters longer than the requested amount of decimal points will be
+    cut off:
+
+    "1.2345"  ->  123 (decimal_points = 2)
+    "10.2030" -> 1020 (decimal_points = 2)
+
+    Params:
+        decimal_points = amount of decimal points to consider
```

Comment:
If `decimal_points` is 0, this will just discard the fractional part, right? If that's correct, how about moving that argument to second place and giving it a default value of 0?
```D
public T floatStringToInt ( T = ulong ) ( cstring value, size_t decimal_points = 0, ReusableException exception = null )
```
The reason for this suggestion is that the main point of the function (as the name says) is to convert a float string to an integer. The intuitively obvious way of doing this is to discard the fractional part. The logic for (essentially) moving the decimal point before doing the conversion seems like an optional extra.